src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterUidTypePass.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\DependencyInjection\CompilerPass;
+
+use Symfony\Bridge\Doctrine\Types\UlidType;
+use Symfony\Bridge\Doctrine\Types\UuidType;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Uid\AbstractUid;
+
+final class RegisterUidTypePass implements CompilerPassInterface
```

Comment:
Shouldn't this class be moved to DoctrineBundle?

Reply:
I put it inside the bridge just were DoctrineValidationPass and RegisterEventListenersAndSubscribersPass are. Also I wouldn't want to have put symfony/uid as a require-dev dependency to the doctrine/bundle. 

I think putting RegisterUidTypesPass [here](https://github.com/doctrine/DoctrineBundle/blob/1.12.x/DoctrineBundle.php#L38) is  sufficient.